src/Form/LocalistEntityForm.php

```diff
@@ -65,11 +69,11 @@ public function form(array $form, FormStateInterface $form_state) {
 
     $form['localist_departments'] = [
       '#type' => 'textfield',
-      '#title' => $this->t('Deparment ID\'s'),
+      '#title' => $this->t('Department ids'),
```

Comment:
Nitpick, but since it's a change... why "ids" instead of "IDs"?. Seems like line 76 should also use "IDs".

Reply:
ids just consistent in all places in the help text. I don't care which it is I just tried to make it consistent. @woodseowl do you have a strong opinion on the way we go?

Reply:
Strong opinion: "IDs" (no apostrophe)